examples/running_flct.py

```diff
@@ -0,0 +1,152 @@
+"""
+===================================
+Fourier Local Correlation Tracking
+===================================
+
+This example applies Fourier Local Correlation Tracking (FLCT)
+to a set of two arrays using `~sunkit_image.flct.flct`.
+"""
+
+import numpy as np
+
+import matplotlib.pyplot as plt
+import sunkit_image.flct as flct
+
+###########################################################################
+# This examples demonstrates how to find the 2D velocity flow field.
+# We will create two dummy images containing a moving object and
+# assume the time difference between the images to be one second.
+# Here we will create two dummy images assuming time between them as
+# 1 second. We also plot both the images as subplots and demonstrate how
+# FLCT functions. This entire example has three parts all of which depicts motion
+# of the image in some particular directions.
```

Comment:
This entire paragraph repeats twice at the start and I don't think it says anything that isn't said elsewhere.